tests/e2e/backup_restore_suite_test.go

```diff
@@ -154,7 +170,7 @@ var _ = Describe("AWS backup restore tests", func() {
 			}
 
 			// wait for pods to be running
-			Eventually(AreAppBuildsReady(dpaCR.Client, brCase.ApplicationNamespace), timeoutMultiplier*time.Minute*3, time.Second*5).Should(BeTrue())
+			Eventually(AreAppBuildsReady(dpaCR.Client, brCase.ApplicationNamespace), timeoutMultiplier*time.Minute*5, time.Second*5).Should(BeTrue())
```

Comment:
Why are we passing the timeout multiplier ? Why not directly pass the timeout ?

Reply:
@weshayutin wanted timeoutMultiplier to re run e2e and see that it's not just a slow cluster issue. It's a configurable multiplier that allow everything to run longer.

Reply:
for debugging failures

Reply:
Allowing some flexibility in extending timeout for slower clusters globally.